examples/Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM python:3.11-slim as builder
 RUN pip install poetry
 COPY . .
 RUN mkdir dist
-RUN touch dist/requirements.txt
-RUN poetry export --without dev --without-hashes --format=requirements.txt | sed '/@ file/d' > dist/requirements.txt
+
+# Generate requirements.txt without local file references
+RUN poetry export --without dev --without-hashes --format=requirements.txt | sed '/^-e /d' > dist/requirements.txt
```

Comment:
Ehh it didn't make a difference but I added it back to the sdk Dockerfile. I reverted all my changes from this examples Dockerfile since we won't be using it anyway